contracts/IERC20.sol

```diff
@@ -1,9 +1,11 @@
+// SPDX-License-Identifier: MIT
+
 // Abstract contract for the full ERC 20 Token standard
 // https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md
-pragma solidity ^0.5.16;
+pragma solidity >=0.7.0 <0.8.0;
 
 
-contract IERC20 {
+interface IERC20 {
```

Comment:
We could get rid of this interface entirely to avoid using the `override public` pattern above. Might be easier that way. But fine with this interface too if it's less work to just leave it.

Reply:
I'm going to leave it for now but ideally we are using the `ERC20` that is in the contracts repo in the future